plugins/outputs/stan/stan.go

```diff
@@ -0,0 +1,152 @@
+package stan
+
+import (
+	"fmt"
+
+	nats_client "github.com/nats-io/go-nats"
+	stan_client "github.com/nats-io/go-nats-streaming"
```

Comment:
I suggest just `nats` and `stan` as package names. This is more idiomatic in Go. If you want to retain the "client" part, then use `natsClient` and `stanClient`.